### PR TITLE
Orc 503 manage homepage testimonies from cms

### PR DESCRIPTION
**Pull request by tiagojsag**

### CODE REVIEW

File: client/src/components/home/carousel.tsx [hunk 1]
```diff
@@ -1,93 +1,69 @@
...
     <motion.div
-      key={card.id}
       className="flex h-[444px] min-w-[373px] max-w-[373px] flex-col justify-between gap-6 border-b-8 border-yellow-500 bg-white p-10 shadow"
       variants={cardVariants}
     >
       <div className="flex h-12 w-12 items-center justify-center bg-gray-700 p-2">
         <Icon className="h-12 w-12 text-white" />
       </div>
-      <p className="text-sm leading-6 text-gray-700">{content}</p>
+      <p
```
Comment (clementprdhomme) [inline client/src/components/home/carousel.tsx:63]: This element must be a `div` otherwise we'll insert `p`s inside of `p`s which is invalid HTML.

**Review by clementprdhomme — COMMENTED** [review]
Some more comments:
- Can we not render the carousel when there aren't any testimony? At the moment, we're still displaying two arrows.
- The wysiwyg content should be limited to about 500 chars (or maybe slightly more for the markup).

**Comment by tiagojsag** [comment]
thanks for the feedback @clementprdhomme , I implemented the changes you requested

**Comment by clementprdhomme** [comment]
Hey @tiagojsag, I think you missed this part 😉:

- Can we not render the carousel when there aren't any testimony? At the moment, we're still displaying two arrows.
- The wysiwyg content should be limited to about 500 chars (or maybe slightly more for the markup).

**Comment by tiagojsag** [comment]
@clementprdhomme done

**Review by clementprdhomme — APPROVED** [review]
CKEditor doesn't currently support limiting the number of characters (https://github.com/ckeditor/ckeditor5/issues/2573) so we're not gonna impose restrictions.

**Pull request merged**